pkg/oc/cli/admin/release/extract.go

```diff
@@ -0,0 +1,86 @@
+package release
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	digest "github.com/opencontainers/go-digest"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+
+	"github.com/openshift/origin/pkg/image/apis/image/docker10"
+	imagereference "github.com/openshift/origin/pkg/image/apis/image/reference"
+	"github.com/openshift/origin/pkg/oc/cli/image/extract"
+)
+
+func NewExtract(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
+	o := &extractOptions{
```

Comment:
Please create appropriate `NewExtractOptions` function, similarly to other commands.